src/govuk/overrides/_spacing.scss

```
@@ -52,7 +56,38 @@ $_spacing-directions: (
   }
 }
 
+/// Generate static spacing override classes
+///
+/// Generate spacing override classes for the given property (e.g. margin)
+/// for each point in the non-responsive spacing scale.
+///
+/// @param {String} $property - Property to add spacing to (e.g. 'margin')
+///
+/// @example css
+///   .govuk-\!-margin-static-4 {
```

Sorry to be nit-picky! I think this should be:

```suggestion
///   .govuk-\!-static-margin-4 {
```

---

ah whoops, fixed that now